docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+version: "3.7"
+
+#    This Source Code Form is subject to the terms of the Mozilla Public License,
+#    v. 2.0. If a copy of the MPL was not distributed with this file, You can
+#    obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+#    the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+#    
+#    Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+#    graphic logo is a trademark of OpenMRS Inc.
+
+services:
+    db:
+        image: mariadb:10.3
+        command: "mysqld --character-set-server=utf8 --collation-server=utf8_general_ci"
+        environment:
+            MYSQL_DATABASE: openmrs
```

Comment:
Does this need to be hardcoded? Or could it default to `openmrs` with ability to override via an environment variable? While we might be assuming "openmrs" for the database schema in some places, we shouldn't be. 🙂

Reply:
I'll make it read an environment variable.